packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec

```diff
@@ -86,9 +81,9 @@ rm %{buildroot}/%{gem_instdir}/Gemfile.lock
 %doc %{gem_instdir}/README.md
 %{gem_instdir}/Rakefile
 %doc %{gem_instdir}/docs
-%doc %{gem_instdir}/pulp_docker_client.gemspec
+%{gem_instdir}/pulp_container_client.gemspec
 %{gem_instdir}/spec
 
 %changelog
-* Fri Jul 5 2019 Justin Sherrill <jlsherrill@gmail.com> 4.0.0b6.dev01562331743-1
-- initial build
+* Mon Dec 02 2019 Justin Sherrill <jlsherrill@gmail.com> 1.0.0rc1-1
+Initial build
```

Comment:
Is this correct?

Reply:
ah, the changelog script generated this part!  i will fix